src/main/java/com/adyen/service/TerminalLocalAPIUnencrypted.java

```diff
@@ -21,12 +21,16 @@
 import java.security.cert.X509Certificate;
 
 /**
+ * @deprecated This class is planned for deprecation in future releases.
```

Comment:
I think a more clear guidance would be: `This class is deprecated and will be removed in on of the next future releases`

Reply:
Thanks. Updated the messaage.